test/unit/specs/select.spec.js

```diff
@@ -701,6 +701,54 @@ describe('Select', () => {
     });
   });
 
+  it('should not open popper when automatic-dropdown not set', done => {
+    vm = createVue({
+      template: `
+        <div>
+          <el-select v-model="value" ref="select">
+            <el-option label="1" :value="1" />
+          </el-select>
+        </div>
+      `,
+      data() {
+        return {
+          value: ''
+        };
+      }
+    }, true);
+
+    vm.$refs.select.$refs.reference.$el.focus();
```

Comment:
reference to input should be `vm.$refs.select.$refs.reference.$refs.input`; $el references to el-input's wrapping `div` which does not focus.

Reply:
Ok. Will implmenet and run the test again.